shared/pkg/config/remote_grype.go

```diff
@@ -23,23 +23,27 @@ import (
 
 const (
 	ScannerRemoteGrypeServerAddress = "SCANNER_REMOTE_GRYPE_SERVER_ADDRESS"
+	ScannerRemoteGrypeServerSchemes = "SCANNER_REMOTE_GRYPE_SERVER_SCHEMES"
 	ScannerRemoteGrypeServerTimeout = "SCANNER_REMOTE_GRYPE_SERVER_TIMEOUT"
 )
 
 type RemoteGrypeConfig struct {
 	GrypeServerAddress string        `yaml:"grype_server_address" mapstructure:"grype_server_address"`
+	GrypeServerSchemes []string      `yaml:"grype_server_schemes" mapstructure:"grype_server_schemes"`
 	GrypeServerTimeout time.Duration `yaml:"grype_server_timeout" mapstructure:"grype_server_timeout"`
 }
 
 func loadRemoteGrypeConfig() RemoteGrypeConfig {
 	setRemoteGrypeScannerConfigDefaults()
 	return RemoteGrypeConfig{
 		GrypeServerAddress: viper.GetString(ScannerRemoteGrypeServerAddress),
+		GrypeServerSchemes: []string{viper.GetString(ScannerRemoteGrypeServerSchemes)},
```

Comment:
I think this should be `viper.GetStringSlice(...)`, in the default you are setting it as a slice and the env var is plural.

Reply:
Fixed.